docs/source/user_guide/index.rst

```diff
@@ -43,6 +43,7 @@ headings and structure.
    smoothing
    centroiding
    spectrum_normalization
+   spectrum_merging
```

Comment:
any reason that the file name is not "spectra_merger"?

Reply:
I think it is better to call it spectrum_merging as it describes the general process (like the other menu items).